include/ddc/kernels/splines/greville_interpolation_points.hpp

```diff
@@ -39,7 +46,12 @@ class GrevilleInterpolationPoints
         using Sampling = ddc::NonUniformPointSampling<tag_type>;
         using SamplingImpl = typename Sampling::template Impl<Kokkos::HostSpace>;
 
-        std::vector<double> greville_points(ddc::discrete_space<BSplines>().nbasis());
+        int n_greville_points = ddc::discrete_space<BSplines>().nbasis();
+        if constexpr (U::is_periodic()) {
+            n_greville_points += 1;
+        }
```

Comment:
How to interpret that the tests pass ? They are not sensitive enough ?

Reply:
It does not answer the question but those changes have been made in sll 4 months ago.

Reply:
There is no test for the length. It is completely broken anyway. In SLL we use a workaround:
https://github.com/gyselax/gyselalibxx/blob/88ab2c66b863ae93297fb10aede7647e52a161ce/src/utils/ddc_helper.hpp#L11-L59

Reply:
Maybe we should now be using periodic sampling?

Reply:
Periodic sampling is currently not in a satisfying condition, there are several strong limitations to its usage in particular https://github.com/CExA-project/ddc/issues/182

Reply:
I agree we should keep PeriodicSampling only for Fourier for now